portal/src/main/webapp/js/src/oncoprint/setup.js

```diff
@@ -151,6 +151,12 @@ var tooltip_utils = {
 		if (d.oncokb_oncogenic) {
 		    ret.append('<img src="images/oncokb-oncogenic-1.svg" title="'+d.oncokb_oncogenic+'" style="height:11px; width:11px;margin-left:3px"/>');
 		}
+		if (d.binary_filter) {
+		    ret.append('<img src="images/button-yellow.png" title="'+d.binary_filter_annotation+'" alt="multi filter" style="height:11px; width:11px;margin-left:3px"/>');
+		}
+		if (d.multi_filter) {
+		    ret.append('<img src="images/button-seagreen.png" title="'+d.multi_filter_annotation+'" alt="multi filter" style="height:11px; width:11px;margin-left:3px"/>');
+		}
```

Comment:
Hovering over the icon for annotation info is not working for me (might be related that the icon itself is not displayed). Also I don’t think its clear you could hover over this. Would it be possible to include annotations in the tooltip between `Mutation:` and `Coverage:` ?

(sorry if I didnt place this comment above the correct section)